content/en/docs/kitex/Tutorials/advanced-feature/error_handler.md

```diff
@@ -1,8 +1,92 @@
 ---
 title: "Customize Error Handler"
-date: 2022-07-20
+date: 2022-07-22
 weight: 4
 description: >
 ---
 
-Translation TODO
+Some users will ask how to let the client-side receive the corresponding error type of the server-side, here to explain, RPC communicates through the protocol, and error handling is also based on the protocol. Usually when the server returns Error, the framework will unify the Error encoding and return it to the client side. If you want the client to return the same error as the server-side, you need to define a set of error codes for handling. However, considering that RPC does not have a unified error code specification, and internal error codes are not necessarily applicable to external users, so the open source part of Kitex does not expose the error code definition, and users can customize their own error handler by using the provided **ErrorHandler**.
+
+## Recommended Usage
+
+ErrorHandler is configured via the client/server Option, but usually, a microservice system will have a unified error handler specification, if you are an enterprise user, it is recommended to customize the Option through the [Suite](...). /... /framework-exten/suite), so that the service developer does not need to pay attention to the configuration of error handler.
```

Comment:
`[Suite](../../framework-exten/suite) `

Reply:
sorry, fixed it and tried, link works fine now.